starter_code/queries.md

```diff
@@ -3,81 +3,141 @@
-
-<!-- Your Code Goes Here -->
+query: 
+projection: 
+sort: { number_of_employees: -1 }
+skip: 
+limit: 10
 
 ### 11. All the companies founded on the second semester of the year. Limit your search to 1000 companies.
-
-<!-- Your Code Goes Here -->
-
-<!-- ### 12. All the companies that have been 'deadpooled' after the third year. -->
-
-<!-- Your Code Goes Here -->
+query: { founded_month: { $gt: 6 } }
+projection: 
+sort: 
+skip: 
+limit: 1000
 
 ### 12. All the companies founded before 2000 that have and acquisition amount of more than 10.000.000
-
-<!-- Your Code Goes Here -->
+query: { $and: [{ founded_year: { $lt: 2000 } }, {"ipo.valuation_amount": { $gt: 10000000} } ] }
+projection: 
+sort: 
+skip: 
+limit:
 
 ### 13. All the companies that have been acquired after 2010, order by the acquisition amount, and retrieve only their `name` and `acquisition` field.
 
-<!-- Your Code Goes Here -->
+query: { "acquisition.acquired_year": {$gt: 2010} }
+projection: { name: 1, acquisition: 1, _id: 0 }
+sort: { "acquisition.amount": -1} 
+skip: 
+limit:
 
 ### 14. Order the companies by their `founded year`, retrieving only their `name` and `founded year`.
 
-<!-- Your Code Goes Here -->
+query: { founded_year: { $gt: 2010 } }
+projection: { name: 1, founded_year: 1, _id: 0 }
+sort: 
+skip: 
+limit:
 
 ### 15. All the companies that have been founded on the first seven days of the month, including the seventh. Sort them by their `acquisition price` descendently. Limit the search to 10 documents.
 
-<!-- Your Code Goes Here -->
+query: { founded_day: { $lte: 7 } }
+projection: 
+sort: { "acquisition.price_amount": -1 }
+skip: 
+limit:
 
 ### 16. All the companies on the 'web' `category` that have more than 4000 employees. Sort them by the amount of employees in ascendant order.
 
-<!-- Your Code Goes Here -->
+query: { $and: [ { category_code: {$eq: "web"} }, { number_of_employees: {$gt: 4000} } ] }
+projection: 
+sort: { number_of_employees: 1 }
+skip: 
+limit:
 
 ### 17. All the companies which their acquisition amount is more than 10.000.000, and currency are 'EUR'.
 
-<!-- Your Code Goes Here -->
+query: { $and: [ { "acquisition.price_amount": {$gt: 10000000} }, { "acquisition.price_currency_code": {$eq: "EUR"} } ] }
+projection: 
+sort: 
+skip: 
+limit:
 
 ### 18. All the companies that have been acquired on the first trimester of the year. Limit the search to 10 companies, and retrieve only their `name` and `acquisition` fields.
 
-<!-- Your Code Goes Here -->
+query: { "acquisition.acquired_month": {$lte: 3} }
+projection: { name: 1, acquisition: 1, _id: 0 }
+sort: 
+skip: 
+limit:
 
 ### 19. All the companies that have been founded between 2000 and 2010, but have not been acquired before 2011.
 
-<!-- Your Code Goes Here -->
+query: { $and: [ { founded_year: {$gte: 2000} }, { founded_year: {$lte: 2010} }, { $or: [ { "acquisition.acquired_year": {$gte: 2011} }, { acquisition: {$exists: false} } ] } ] }
+projection: 
+sort: 
+skip: 
+limit:
```

Comment:
De manière plus simple, tu peux faire :
`{ $and: [ { founded_year: {$gte: 2000} }, { founded_year: {$lte: 2010} }, {"acquisition.acquired_year": {$gte: 2011}}]}`